src/axom/inlet/docs/sphinx/quick_start.rst

```diff
@@ -0,0 +1,124 @@
+they have to present in the given input deck. You can also set a default value to each field via the type-safe
+``Field::defaultValue()`` member functions. Doing so will populate the corresponding Fields value
+if the specific Field is not present in the input deck. The following example shows these concepts:
+
+.. literalinclude:: ../../examples/verification.cpp
+   :start-after: _inlet_workflow_defining_schema_start
+   :end-before: _inlet_workflow_defining_schema_end
+   :language: C++
+
+
+.. _inlet_verification_label:
+
+Verification
+------------
+
+This step helps ensure that the given input deck follows the rules expected by the code.  These
+rules are not verified until you call ``Inlet::verify()``.  Doing so will return true/false and
+output SLIC warnings to indicate which Field or Table violated which rule.
+
+As shown above, both Tables and Fields can be marked as ``required``. Fields have two additional
+basic rules that can be enforced with the following ``Field`` class member functions:
+
+========================= ===================
+Name                      Description
+========================= ===================
+validValues               Indicates the Field can only be set to one of the given values.
+range                     Indicates the Field can only be set to inclusively between two values.
+========================= ===================
+
+Inlet also provides functionality to write your own custom rules via callable lambda verifiers.
+Fields and Tables can both register one lambda each via their ``registerVerifier()`` member functions.
+The following example adds a custom verifier that simply verifies that the given ``dimensions`` field
+match the length the given vector:
+
+.. literalinclude:: ../../examples/verification.cpp
+   :start-after: _inlet_workflow_verification_start
+   :end-before: _inlet_workflow_verification_end
+   :language: C++
+
+.. note::  ``Inlet::getGlobalTable()->registerVerifier()`` can be used to add a verifier to apply rules
+  to the Fields at the global level.
+
+
+.. _inlet_accessing_data_label:
+
+Accessing Data
+--------------
+
+After the input deck has been read and verified by the previous steps, you can access the data by name
+via ``Inlet::get()`` functions.  These functions are type-safe, fill the given variable with what is found,
+and return a boolean whether the Field was present in the input deck or had a default value it could fall
+back on.  Variables, on the Inlet side, are used in a language-agnostic way and are then converted 
+to the language-specific version inside of the appropriate ``Reader``. For example, Inlet refers to the
+Lua variable ``vector={x=3}`` or ``vector.x`` as ``vector/x`` on all Inlet function calls.
+
+For example, given the previous verificiation example, this access previously read values:
+
+.. literalinclude:: ../../examples/verification.cpp
+   :start-after: _inlet_workflow_accessing_data_start
+   :end-before: _inlet_workflow_accessing_data_end
```

Comment:
Consider showing both `vector.x` and vector/x` notations in this example
(with comments to indicate that these are two different ways to access the data)?

e.g.
```cpp
  x_found = myInlet->get("vector/x", x);   // you can access element using '/' as a separator
  y_found = myInlet->get("vector.y", y);   // you can also use '.' as a separator
```

Reply:
you can't use vector.y at this point everything must be language agnostic